relay-quotas/src/lib.rs

```diff
@@ -11,9 +11,14 @@
 /// typically happens for disabled keys, projects, or organizations.
 const REJECT_ALL_SECS: u64 = 60;
 
+#[cfg(feature = "redis")]
```

Comment:
Would move this together with:

```rs
#[cfg(feature = "redis")]
pub use self::global::*;
```

Directly before the `mod redis` block. All `redis` features together and alphabetically sorted.